### PR TITLE
Sig 4451 add photo kto discontent

### PR DESCRIPTION
**Pull request by OscarBakker**
## Signalen

**to reviewers**
[SIG-4582-change-trans-contac](https://github.com/Amsterdam/signals-frontend/tree/SIG-4582-change-trans-contac)
needs to be reviewed first, then we can merge it, and set base to develop.

 https://datapunt.atlassian.net/browse/SIG-4451?atlOrigin=eyJpIjoiODRhNWVkZjM2YTMxNDMzYjkyMWZkOWJjZWI4NmI3NmMiLCJwIjoiaiJ9

**for testers**
confirm that kto is uploading by checking the network tab
test if its still working, showing the file upload only when satisfaction equals no

Before opening a pull request, please ensure:

- [x] Double-check your branch is based on `develop` and targets `develop`
- [x] Pull request has tests (we are going for 100% coverage!)
- [x] Code is well-commented, linted and follows project conventions
- [x] Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/incident/containers/KtoContainer/components/KtoForm/index.js [hunk 1]
```diff
@@ -215,23 +218,57 @@ const KtoForm = ({ options, isSatisfied, onSubmit }) => {
...
+            <StyledLabel htmlFor="text_extra">
+              {"Foto's toevoegen? "}
+              <Optional>(niet verplicht)</Optional>
+            </StyledLabel>
+            <HelpText id="subtitle-kto">
+              Voeg een foto toe om de situatie te verduidelijken.
+            </HelpText>
+            <FileInput
+              handler={() => {}}
+              parent={{ meta: { updateIncident: () => {} } }}
```
Comment (OscarBakker) [inline src/signals/incident/containers/KtoContainer/components/KtoForm/index.js:233]: empty, do we need to update the incident when in KTO? The incident seems to be final, so leave this empty for now.

File: src/signals/incident/containers/KtoContainer/components/KtoForm/index.js [hunk 2]
```diff
@@ -215,23 +218,57 @@ const KtoForm = ({ options, isSatisfied, onSubmit }) => {
...
+              meta={{
```
Comment (OscarBakker) [inline src/signals/incident/containers/KtoContainer/components/KtoForm/index.js:234]: these meta fields I took over from wizard-1, might just make maxNumberOfFiles === 1

**Comment by OscarBakker** [comment]
Screenshot 2022-06-01 at 08 51 59
first step

**Comment by OscarBakker** [comment]
waiting for signal id

**Review by kiruys — APPROVED** [review]
looks good, and gives me some inspiration for my own tests...

**Pull request merged**